pytest_watch/watcher.py

```diff
@@ -164,8 +173,11 @@ def watch(directories=[], ignore=[], auto_clear=False, beep_on_failure=True,
         observer.join()
     except KeyboardInterrupt:
         observer.stop()
-    if onexit:
-        os.system(onexit)
+        if onsigint:
+            os.system(onsigint)
+    else:
+        if onexit:
+            os.system(onexit)
```

Comment:
Ok cool. Thanks!

I'm currently rewriting some of the logic so that only one instance of py.test gets run at a time. Just wanted to make sure I'm not stepping on your toes with a specific case I didn't know about.